packages/lab/src/accordion/AccordionHeader.css

```diff
@@ -0,0 +1,42 @@
+.saltAccordionHeader {
+  display: flex;
+  gap: var(--salt-spacing-100);
+  padding-block: var(--salt-spacing-50);
+  padding-inline: var(--salt-spacing-100);
+  align-items: center;
+  border: 0;
+  width: 100%;
+  min-height: var(--salt-size-base);
+  background: var(--salt-actionable-secondary-background);
+  font-size: var(--salt-text-fontSize);
+  line-height: var(--salt-text-lineHeight);
+  font-family: var(--salt-text-fontFamily);
+  color: var(--salt-text-primary-foreground);
+  --saltIcon-color: var(--salt-text-primary-foreground);
+  text-align: left;
+}
+
+.saltAccordionHeader:focus-visible {
+  background: var(--salt-actionable-primary-background-hover);
+  outline: var(--salt-focused-outline);
+}
+
+@media (hover: hover) and (pointer: fine) {
```

Comment:
This is to prevent the hover style on mobile press? We don't do this in other places. Should we start?

Reply:
But then again, you don't have an :active style. Does that mean the mobile user will see no visual indicator of their press?

Reply:
We should probably do it elsewhere but this is because of a lack of an `:active` style. Accordion wasn't designed with one